paket.dependencies

```diff
@@ -38,6 +38,8 @@ nuget Topshelf.FSharp
 nuget fszmq
 nuget ApacheThrift
 nuget elmah.io.client
+nuget Google.Logging.V2 1.0.0-beta02
```

Comment:
The older version is on purpose. Starting with beta04, there's an API change in the Gax library that breaks clients.  Google has moved from IPagedEnumerable<TPage, TItem> to PagedEnumerable<TPage, TItem> and they haven't rolled those APIs out everywhere.